client/my-sites/site-settings/form-analytics.jsx

```diff
@@ -130,7 +130,7 @@ export class GoogleAnalyticsForm extends Component {
 					siteIsJetpack
 						? translate( "Monitor your site's views, clicks, and other important metrics" )
 						: translate(
-								"Add your unique tracking ID to monitor your site's performance in Google Analytics."
+								"Add your unique Measurement ID to monitor your site's performance in Google Analytics."
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.